clouddriver-kubernetes/src/main/groovy/com/netflix/spinnaker/clouddriver/kubernetes/v1/api/KubernetesClientApiAdapter.groovy

```diff
@@ -144,14 +146,18 @@ class KubernetesClientApiAdapter {
       /*
        "fixme" and note this is k8s-client api issue and we are working this as workaround.
         */
-      V1beta1StatefulSetList list = apiInstance.listNamespacedStatefulSet(namespace, null, null, null, null, API_CALL_TIMEOUT_SECONDS, false)
-      String apiVersion = list.getApiVersion();
-      for (V1beta1StatefulSet item : list.getItems()) {
-        item.setApiVersion(apiVersion);
-        item.setKind("StatefulSet");
+      try {
+        V1beta1StatefulSetList list = apiInstance.listNamespacedStatefulSet(namespace, null, null, null, null, API_CALL_TIMEOUT_SECONDS, false)
+        String apiVersion = list.getApiVersion();
+        for (V1beta1StatefulSet item : list.getItems()) {
+          item.setApiVersion(apiVersion);
+          item.setKind("StatefulSet");
+        }
+
+        return list.items
+      } catch (ApiException ex) {
+        log.info("Cannot fetch Statefulsets [Required Kubernates VERSION>=1.6")
```

Comment:
Unterminated `[` in log statement

This is going to be very noisy for users who don't care about stateful sets. I'd put this in `.debug`

Reply:
s/Kubernates/Kubernetes/g